### ISSUE TITLE
Sidestep gzip?

### ISSUE THREAD

**Issue by rclark** [mapbox/vtinfo#12]
I often get a file that _might_ be a vector tile, or maybe I'm making tiles with some library and I'm not sure if I botched it or not. Its nice to be able to just `vtinfo some-file` to find out if its a valid tile.

However sometimes the files are **gzipped** vector-tiles. Could `vtinfo` detect this and gunzip prior to info-ing?

cc @mapsam

**Comment by springmeyer** [comment]
Yes, I think it would make sense for vtinfo to automatically support either uncompressed mvt or gzip compressed mvt. node-mapnik/mapnik-vt uses this behavior. https://github.com/mapbox/cpp/issues/10 is a quick prerequisite to getting this going for vtinfo.
